runtime/compiler/optimizer/IdiomRecognitionUtils.cpp

```diff
@@ -892,16 +892,17 @@ createArrayAddressTree(TR::Compilation *comp, bool is64bit, TR::Node *baseNode,
    {
    TR::Node *top = NULL;
    TR::Node *c2 = NULL;
+   TR::Node *aload = createLoad(baseNode);
    if (indexNode->getOpCodeValue() == TR::iconst && indexNode->getInt() == 0)
       {
-      top = TR::TransformUtil::generateArrayStartTrees(comp, baseNode);
+      top = TR::TransformUtil::generateArrayStartTrees(comp, aload);
       }
    else
       {
       // c2 = createBytesFromElement(comp, is64bit, indexNode, multiply);
       // createBytesFromElement esentially creates offset trees
```

Comment:
We don't need to keep these comment lines around. Can you please remove those as well.